### PR TITLE
Install npm versions of origami packages

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

File: assets/js/lib/faq.js [hunk 1]
```diff
@@ -1,105 +1,105 @@
...
-				window.scrollTo(0, dtCoordinate.top - stickyHeaderCompensation )
-			}
-		}, 500);
-
-	}
-});
+// 'use strict';
+//
+// import Delegate from 'ftdomdelegate';
+// const dom = require('o-dom');
```
Comment (JakeChampion) [inline assets/js/lib/faq.js:4]: o-dom hasn't existed for atleast 2 years, we can use browser APIs instead such as [Element.closest](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest) instead

File: assets/js/lib/faq.js [hunk 2]
```diff
@@ -1,105 +1,105 @@
...
-				const stickyHeaderCompensation = 60;
-				window.scrollTo(0, dtCoordinate.top - stickyHeaderCompensation )
-			}
-		}, 500);
-
-	}
-});
+// 'use strict';
+//
+// import Delegate from 'ftdomdelegate';
```
Comment (JakeChampion) [inline assets/js/lib/faq.js:3]: We can use this via npm instead of bower --> https://www.npmjs.com/package/ftdomdelegate

File: gulpfile.js [hunk 3]
```diff
@@ -40,143 +39,143 @@ function generateRandomString(length) {
 	return text;
 }
 
-const alphavilleBuildTools = function (gulp, config) {
```
Comment (JakeChampion) [inline gulpfile.js:43]: We 'unrolled' this function as it was making it a bit hard to read and understand -- I can revert this back if that is wanted

File: package.json [hunk 4]
```diff
@@ -5,15 +5,26 @@
   "scripts": {
     "postinstall": "gulp",
     "start": "node --max-http-header-size=80000 ./bin/www",
-    "prepare": "npx snyk protect || npx snyk protect -d || true"
+		"prepare": "npx snyk protect || npx snyk protect -d || true"
```
Comment (chee) [inline package.json:8]: looks like this got changed from spaces to tabs
```suggestion
    "prepare": "npx snyk protect || npx snyk protect -d || true"
```

**Pull request closed without merging**